sdk/keyvault/keyvault-certificates/src/index.ts

```diff
@@ -884,6 +921,24 @@ export class CertificateClient {
     const requestOptions = operationOptionsToRequestOptionsBase(options);
     const span = this.createSpan("updateIssuer", requestOptions);
 
+    const generatedOptions: KeyVaultClientSetCertificateIssuerOptionalParams = requestOptions;
+
+    if (
+      options.organizationId ||
+      (options.administratorContacts && options.administratorContacts.length)
+    ) {
+      generatedOptions.organizationDetails = {
+        id: options.organizationId,
+        adminDetails: options.administratorContacts
+      };
+    }
+
+    if (options.enabled) {
+      generatedOptions.attributes = {
```

Comment:
Are the other attributes not available here?

Reply:
The other attributes are read-only on the core code 🌞

Reply:
So they're not available to use to pass along to the generatedOptions?

Reply:
I will double check in other languages!

Reply:
Checked, updatedOn and createdOn are not useful if sent to the service.

Reply:
Yeah, only the service sets those. May even fail if you do send them, or maybe just ignore them.

Reply:
Should be good now :)